coconut/configuration/configuration.go

```diff
@@ -391,9 +397,35 @@ func Import(cfg *configuration.ConsulSource, cmd *cobra.Command, args []string,
 		return err, logicError
 	}
 
+	// Temporary workaround to allow no-versioning
+	latestTimestamp, err := componentcfg.GetLatestTimestamp(keys, component, entry)
+	if err != nil {
+		return err, invalidArgs
+	}
+
+	if entryExists {
+		if latestTimestamp != "0" && useNoVersion {
+			// If a timestamp already exists in the entry specified by the user, than it cannot be used
+			return errors.New("Specified entry: '" + entry + "' already contains versioned items. Please " +
+				"specify a different entry name"), invalidArgs
+		}
+		if latestTimestamp == "0" && !useNoVersion {
+			// If a timestamp does not exist for specified entry but user wants versioning than an error is thrown
+			return errors.New("Specified entry: '" + entry + "' already contains un-versioned items. Please " +
+				"specify a different entry name"), invalidArgs
+		}
+	}
```

Comment:
Yes very good this is exactly what I had in mind. Versioned and unversioned entries we simply don't mix.